helm/stacks-blockchain/README.md

```diff
@@ -0,0 +1,136 @@
+| node.replicaCount | Number of stacks-blockchain nodes to run | 1 |
+| node.rpcPort | Port used for RPC connections | 20443 |
+| node.p2pPort | Port used for P2P connections | 20444 |
+| node.labels | Labels to be added to the node Deployment | {} |
+| node.annotations | Annotations to be added to the node Deployment | {} |
+| node.podAnnotations | Annotations to be added to the node pod(s) | {} |
+| node.podAnnotationConfigChecksum | Add Config Checksum to pod Annotations.<br>This will trigger Rolling Updates on configuration changes. | false |
+| node.podSecurityContext | Security Context policies for the node pod(s) | {} |
+| node.securityContext | Security Context policies for the node container | {} |
+| node.debug | Set to true for verbose logging. Set to false for normal verbosity | false |
+| node.jsonLogging | Set to true to enable json logging. Set to false for normal logging | false |
+| node.command | Overriding this is typically not necessary, unless you wish to run the node in a different way | ["/bin/stacks-node"] |
+| node.args | Overriding this is typically not necessary, unless you wish to run the node in a different way | ["start", "--config", "/src/stacks-node/config.toml"] |
+| node.extraEnv | Extra environment variables to add to the node's container | [] |
+| node.resources | Specify resource limits and requests for the node's container | {} |
+| node.revisionHistoryLimit | Rollback limit | 10 |
+| node.updateStrategy | The update strategy to apply | {} |
+| node.minReadySeconds | minReadySeconds to avoid killing pods before it's ready | 0 |
+| node.nodeSelector | Node labels for node pod assignment | {} |
+| node.tolerations | Node tolerations for server scheduling to nodes with taints | {} |
+| node.affinity | Affinity and anti-affinity | {} |
+| node.terminationGracePeriodSeconds | Total time it takes for the Container to stop normally | 60 |
+| node.volumes | Additional volumes for the node | [] |
+| node.volumeMounts | Additional volumeMounts for the node | [] |
+| node.extraContainers | Additional containers to run alongside the node. Useful if adding a sidecar | [] |
+| node.initContainers | Containers which are run before the node container is started | [] |
+| config | More configs can be added than what's shown below.All children fields under the node, burnchain, and mstx_balance fields will be converted from YAML to valid TOML format in the Configmap.<br><br>For info on more available config fields, please reference to our [example config files located here](https://github.com/blockstack/stacks-blockchain/tree/master/testnet/stacks-node/conf). |  |
+| config.node.rpc_bind |  | 0.0.0.0:20443 |
+| config.node.p2p_bind |  | 0.0.0.0:20444 |
+| config.node.seed | Replace with your private key if deploying a miner node | nil |
+| config.node.miner | Set this to true if deploying a miner node.<br>Set this to true if deploying a follower node. | false |
```

Comment:
typo - one of these bools should not be `true`. i suspect the follower mode should be `false`?